### PR TITLE
fix issue #19430

### PR DESCRIPTION
**Pull request by xyq175com**
This PR fix issue #19430(Incorrect classification example usage for Linux (azurerm_automation_software_update_configuration)]

### CODE REVIEW

File: website/docs/r/automation_software_update_configuration.html.markdown [hunk 1]
```diff
@@ -49,160 +49,160 @@ resource "azurerm_automation_software_update_configuration" "example" {
...
 The `timeouts` block allows you to specify [timeouts](https://www.terraform.io/docs/configuration/resources.html#timeouts) for certain actions:
 
-* `create` - (Defaults to 30 minutes) Used when creating the Automation.
-* `read` - (Defaults to 5 minutes) Used when retrieving the Automation.
-* `update` - (Defaults to 10 minutes) Used when updating the Automation.
-* `delete` - (Defaults to 10 minutes) Used when deleting the Automation.
+- `create` - (Defaults to 30 minutes) Used when creating the Automation.
+- `read` - (Defaults to 5 minutes) Used when retrieving the Automation.
+- `update` - (Defaults to 10 minutes) Used when updating the Automation.
+- `delete` - (Defaults to 10 minutes) Used when deleting the Automation.
```
Comment (stephybun) [inline website/docs/r/automation_software_update_configuration.html.markdown:205]: These don't need to/shouldn't be replaced with dashes, could you set these back?

**Review by stephybun — CHANGES_REQUESTED** [review]
Thanks for this @xyq175com. One minor remark otherwise this is good to go.

File: website/docs/r/automation_software_update_configuration.html.markdown (hunk 1 above)
Comment (xyq175com) [reply website/docs/r/automation_software_update_configuration.html.markdown]: > These don't need to/shouldn't be replaced with dashes, could you set these back?
Its weird, I used vs code to make change, and once I change from dash to star, and save it it auto save as dash...
Anyway I just changed these back via online editor. Please review.

**Review by stephybun — APPROVED** [review]
Thanks @xyq175com LGTM 🍈

**Pull request merged**